### PR TITLE
use miniforge optionally

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: deps/build.jl [hunk 1]
```diff
@@ -13,6 +13,9 @@ module DefaultDeps
     if !isdefined(@__MODULE__, :ROOTENV)
         const ROOTENV = joinpath(Main.condadir, MINICONDA_VERSION)
     end
+    if !isdefined(@__MODULE__, :USE_MINIFORGE)
+        const USE_MINIFORGE = "0"
```
Comment (stevengj) [inline deps/build.jl:17]: I guess we might eventually make this the default, but I agree that it should be experimental to start with.

Comment (stevengj) [reply deps/build.jl:17]: Seems like it should be a boolean, though.

File: deps/build.jl [hunk 2]
```diff
@@ -27,6 +30,8 @@ ROOTENV = get(ENV, "CONDA_JL_HOME") do
     end
 end
 
+USE_MINIFORGE = get(ENV, "CONDA_JL_USE_MINIFORGE", DefaultDeps.USE_MINIFORGE)
```
Comment (stevengj) [inline deps/build.jl:33]: To use a boolean,
```jl
USE_MINIFORGE = lowercase(get(ENV, "CONDA_JL_USE_MINIFORGE", DefaultDeps.USE_MINIFORGE ? "1" : "0")) in ("1","true","yes")
```

**Comment by stevengj** [comment]
LGTM; needs an update to the docs.

**Comment by stevengj** [comment]
Can we add it to the test matrix?

**Comment by DilumAluthge** [comment]
Also probably a good time to switch to GitHub Actions for CI?

**Comment by isuruf** [comment]
@DilumAluthge, feel free to push to this branch if you have commit rights. (or a PR if not)

**Review by stevengj — APPROVED** [review]

**Pull request merged**